generator/types.go

```diff
@@ -651,6 +657,25 @@ func (f *Field) TypeSchemaName() string {
 	return parts[len(parts)-1]
 }
 
+func (f *Field) mustFindArrayLenght() int {
```

Comment:
In fact, you can avoid using regexps at all: https://play.golang.org/p/lZNoERWxeb

```go
func arrayLen(f *Field) (int, error) {
	if f.Kind != Array {
		return 0, fmt.Errorf("field %s is not an array", f.Name)
	}
	
	idx := strings.Index(f.Type, "]")
	return strconv.Atoi(f.Type[1:idx])
}
```